api/app/signals/apps/api/v1/views/pdf.py

```diff
@@ -101,10 +151,13 @@ def get_context_data(self, **kwargs):
                 rd_coordinates.x + 340.00,
                 rd_coordinates.y + 125.00,
             )
+        jpg_data_urls = self._get_context_data_images(self.object)
+
         return super(GeneratePdfView, self).get_context_data(
             bbox=bbox,
             img_data_uri=img_data_uri,
-            images=self.object.attachments.filter(is_image=True),
+            # images=self.object.attachments.filter(is_image=True),
```

Comment:
Remove this instead of commenting it, also if we are going to use the `jpg_data_urls` we can remove the images from the template.

Reply:
Will do